zhaquirks/ikea/__init__.py

```diff
@@ -73,6 +82,94 @@ class ScenesCluster(CustomCluster, Scenes):
     )
 
 
+class ShortcutV1Cluster(EventableCluster):
+    """Ikea Shortcut Button Cluster Variant 1."""
+
+    name = "ShortCutV1EventsCluster"
+    cluster_id = IKEA_SHORTC_CLUSTER_V1_ID
+
+    server_commands = {
+        0x01: foundation.ZCLCommandDef(
+            "ShortCutV1Events",
+            {
+                "Shortcut_Button": t.int8s,
+                "Shortcut_Event": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+    }
+
+
+class ShortcutV2Cluster(EventableCluster):
+    # class ShortcutV2Cluster(OnOff, EventableCluster):
+    """Ikea Shortcut Button Cluster Variant 2."""
+
+    name = "ShortcutClusterV2"
+    cluster_id = IKEA_M_SWITCH_CLUSTER_ID
+
+    server_commands = {
+        0x00: foundation.ZCLCommandDef(
+            "switch_latched",
+            {
+                "NewPosition": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+        0x01: foundation.ZCLCommandDef(
+            "initial_press",
+            {
+                "NewPosition": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+        0x02: foundation.ZCLCommandDef(
+            "long_press",
+            {
+                "PreviousPosition": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+        0x03: foundation.ZCLCommandDef(
+            "short_release",
+            {
+                "PreviousPosition": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+        0x04: foundation.ZCLCommandDef(
+            "long_release",
+            {
+                "PreviousPosition": t.int8s,
+            },
+            False,
+            is_manufacturer_specific=True,
+        ),
+        0x05: foundation.ZCLCommandDef(
+            "multi_press_ongoing",
+            {
+                "NewPosition": t.int8s,
+                #                 "CurrentNumberOfPressesCounted": t.int8s, # Not implanted by IKEA for the moment.
```

Comment:
I'm guessing this is something Matter-related that IKEA didn't implement?
I think the comment is fine, but remove the extra spaces after `#`